Junk Codes/Ultrasonic.ino

```diff
@@ -1,138 +1,124 @@
-#include <Servo.h>
-#include <PubSubClient.h>
-#include <WiFi.h> //#include <ESP8266WiFi.h>
-#include <WiFiClient.h>
-#include <WiFiServer.h>
-#include <WiFiUdp.h>
-#include <SoftwareSerial.h> //check
-char *ssid = "JAILAVANYA";
-char *password = "imrich325@@DRDO";
 
-const char *mqttServer = "192.168.0.103";
+#include<ESP8266WiFi.h>
+#include<NewPing.h>
+#include<Stepper.h>
+
+const char *ssid = "JAILAVANYA";
+const char *password = " ";
+
+#include<PubSubClient.h>
+
+const char *mqttServer = "192.168.0.108";
 const int mqttPort = 1883;
-const char *mqttUser = "mqtt";
-const char *mqttPassword = "mqtt";
+const char *mqttUser = "mqtt2020";
+const char *mqttPassword = "mqtt2020";
 const char *mqttClientName = "ultrasonic sensor";
+WiFiClient ULTRASONICSENSOR;
+PubSubClient client(ULTRASONICSENSOR);
 
-WiFiClient ultrasonicsensor;
-PubSubClient client(ultrasonicsensor);
+#define ultrangle "home/deg"
+#define ultradist "home/dist"
+#define mqttRadar "home/radarData"
 
-#define mqttTempExt "home/latExt" //change topic and variable
-#define mqttHumExt "home/lngExt"  //change topic and variable
-const int trigPin = 9;
-const int echoPin = 10;
+const float STEPS_PER_REV = 32;
+const float GEAR_RED = 64;
+const float STEPS_PER_OUT_REV = STEPS_PER_REV * GEAR_RED;
+int StepsRequired = 2048;
+Stepper steppermotor(STEPS_PER_REV,8,10,9,11);
 long duration;
-int distinCM;
-Servo radarServo;
-float lngExt;           //------
-float latExt;           //------
-const char *mqttLatExt; //-------
-const char *mqttLngExt; //-------
+int distance;
+const int trigPin = 9;
+const int echoPin = 10; 
+int deg,dist;
+String radarData;
 
 void setup()
 {
-  pinMode(trigPin, OUTPUT);
-  pinMode(echoPin, INPUT);
-  Serial.begin(9600);
-  radarServo.attach(11);
+pinMode(trigPin, OUTPUT);
+pinMode(echoPin, INPUT);
+Serial.begin(9600);
+Serial.begin(115200);
+Serial.println();
+connect();
+}
 
-  // void connectWifi
+void connect()
+{
   Serial.print("Connecting to ");
-  Serial.println(ssid);
-  //  WiFi.mode(WIFI_STA);
-  WiFi.disconnect();
-  delay(2000);
-  WiFi.begin(ssid, password);
-
-  while (WiFi.status() != WL_CONNECTED)
-  {
-    delay(500);
-    Serial.print(".");
-  }
-  Serial.println("");
-  Serial.println("WiFi connected");
-  Serial.println("IP address: ");
-  Serial.println(WiFi.localIP());
+Serial.println(ssid);
+WiFi.mode(WIFI_STA);
+WiFi.disconnect();
+delay(2000);
+WiFi.begin(ssid, password);
+while (WiFi.status() != WL_CONNECTED)
+{
+delay(500);
+Serial.print("=");
+}
+Serial.println("");
+Serial.println("WiFi connected");
+Serial.println("IP address: ");
+Serial.println(WiFi.localIP());
+client.setServer(mqttServer, mqttPort);
```

Comment:
shift this line `client.setServer(mqttServer, mqttPort);` to `void setup` because this is not the part of wifi connect